examples/gitlab-asg/Makefile

```diff
@@ -61,23 +60,61 @@ plan:
 apply:
 	@terraform apply tf.out
 
+## Generate the ssh_congig both for Git operations and instance maintenance
+render-ssh-config:
+	@. ./vars.env && envsubst <ssh_config.tpl > ssh_config
+	@echo `aws ec2 describe-instances --filters "Name=tag:Name,Values=at-test-gitlab-server-eu-central-1a" --region=eu-central-1 | jl _.Reservations[0].Instances[0].PublicIpAddress | tr -d '"'` >> ssh_config
+
 ## use ops cli tool to lookup the public IP of the gitlab instance
-get-ip:
-	@ops aws ec2 asg ips -r $$(terraform output region) $$(terraform output gitlab_asg_name)
+##get-ip:
+##	@ops aws ec2 asg ips -r $$(terraform output region) $$(terraform output gitlab_asg_name)
+
+## SSH to the gitlab server and use parted and similar tools to setup a partition and filesystem on the EBS volume (run only once per volume)
+init-ebs-volume:
+	@ssh data-ops-gitlab "sudo parted --script /dev/xvdf -- mklabel msdos && \
```

Comment:
Since this target doesn't currently work, I think performing these commands on the instance manually should be documented in the README.

Reply:
I think a better approach is to remove them from the Makefile at this point. They aren't currently in Master so it's not a regression. The whole point is to get in a bunch of fixes that have been sitting around for a very long time into `master`. I will save these commands and come back to them in next one or two MRs.

Reply:
Fixed as per above

Reply:
I do think it would be good to document the need to run these commands in the README.  Otherwise, you lose all your data if the instance is ever replaced by the ASG (which is, after all, the whole point of having the ASG).